### PR TITLE
Fix indentation [ci skip]

### PR DESCRIPTION
**Pull request by ydakuka**

### CODE REVIEW

File: guides/source/active_support_core_extensions.md [hunk 1]
```diff
@@ -1989,7 +1989,7 @@ Extensions to `BigDecimal`
 The method `to_s` provides a default specifier of "F". This means that a simple call to `to_s` will result in floating point representation instead of engineering notation:
 
 ```ruby
-BigDecimal.new(5.00, 6).to_s  # => "5.0"
+BigDecimal.new(5.00, 6).to_s       # => "5.0"
```
Comment (kamipo) [inline guides/source/active_support_core_extensions.md]: Is this change needed?

Comment (ydakuka) [inline guides/source/active_support_core_extensions.md]: See [before](https://github.com/rails/rails/blob/master/guides/source/active_support_core_extensions.md#to_s) and [after](https://github.com/ydakuka/rails/blob/566a056115d207fe2c83e7131bc196558331958d/guides/source/active_support_core_extensions.md#to_s)
